activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   Use `extract_content_type_from_io: false` as an argument in
+    `ActiveStorage::Blob` `#create_after_upload` and `#build_after_upload` to
+    manually override the automatic extraction of content type from the io.
+
+    This address the issue where the user defines a content type as an argument
+    but the automatic MimeType inference from the io is not what is expected.
+    It means a user avoids needing force the correct content type by
+    patching `ActiveStorage::Blob.extract_content_type`.
```

Comment:
It’s uncommon to call `ActiveStorage::Blob#create_after_upload` or `#build_after_upload` directly. Can we demonstrate passing `identify: false` to `ActiveStorage::Attached::{One,Many}#attach` instead?

Reply:
I've added the update but not sure if the content is what you had in mind. It has a similar wording to the comments in blob.rb and I've added in an example based on what is in the RailsGuides. 

commit 190693c